src/llv8.cc

```diff
@@ -201,12 +201,24 @@ uint32_t JSFrame::GetSourceForDisplay(bool reset_line, uint32_t line_start,
   return line_start + lines_found;
 }
 
+// On 64 bits systems, V8 stores SMIs (small ints) in the top 32 bits of
```

Comment:
Nit: please use double newline between functions. It is not enforced by linter, but I prefer this style.